src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(value = 0, n = 0) {
+      return value + n;
+    },
+
+    subtract(value = 0, n = 0) {
+      return value - n;
+    },
+
+    multiply(value = 0, n = 1) {
+      return value * n;
+    },
+
+    divide(value = 0, n = 1) {
+      return value / n;
+    },
```

Comment:
You need to add a check by 0, because according to the rules of mathematics, you cannot divide by 0